extension/goplugin/transaction_test.go

```diff
@@ -68,7 +68,12 @@ var _ = Describe("Transaction", func() {
 			testSchema = env.Schemas().Find("test")
 			Expect(testSchema).To(Not(BeNil()))
 
-			err = db.InitDBWithSchemas(DbType, DbFile, true, true, false)
+			err = db.InitDBWithSchemas(DbType, DbFile, db.InitDBParams{
```

Comment:
there's some suspicious inconsistency here: in some tests it's true/false/false/true, in others false/false/false/true and here it's true/true/false/true. why is that?
could we add some universal safe defaults? eg.
```
func DefaultTestDBParams() InitDbParams
```

Reply:
ok, fixed